src/frequenz/client/electricity_trading/_client.py

```diff
@@ -13,10 +13,12 @@
 # pylint: disable=no-member
 from frequenz.api.electricity_trading.v1 import electricity_trading_pb2
 from frequenz.api.electricity_trading.v1.electricity_trading_pb2_grpc import (
+    ElectricityTradingServiceAsyncStub,
```

Comment:
@llucax, we can't import the async stub like this, right?  Should this only be under `if typing.TYPE_CHECKING:`?

Reply:
This was done [here](https://github.com/frequenz-floss/frequenz-client-base-python/releases/tag/v0.7.0) in the example section of the upgrading section of the new base client release. But since it's just an example I don't know if it was tested

Reply:
I just tried as well, can't find an easy solution.  I guess we need to wait for @llucax

Reply:
Thanks @Marenz for the summary, that should work.

I just updated the release notes (once again) to hopefully provide instructions that finally work.

Reply:
Correction: I wrote `electricity_trading_pb2` but meant `electricity_trading_pb2_grpc`, it is updated in my comment

Reply:
I still get an error from `pytest_min`: 
```
ERROR AttributeError: module 'frequenz.api.electricity_trading.v1.electricity_trading_pb2_grpc' has no attribute 'ElectricityTradingServiceAsyncStub'
```